src/main/java/org/springframework/hateoas/MediaTypes.java

```diff
@@ -34,4 +35,14 @@
 	 * Public constant media type for {@code application/hal+json}.
 	 */
 	public static final MediaType HAL_JSON = MediaType.valueOf(HAL_JSON_VALUE);
+	
+	/**
+	 * A String equivalent of {@link MediaTypes#HAL_JSON_UTF8}.
+	 */
+	public static final String HAL_JSON_UTF8_VALUE = HAL_JSON_VALUE + ";charset=UTF-8";
```

Comment:
I agree about the String concatenation, but it's interesting you mention `MediaType` because that's where I modeled my `hal+json` additions from. See [this line](https://github.com/spring-projects/spring-framework/blob/master/spring-web/src/main/java/org/springframework/http/MediaType.java#L103)

Also it uses `UTF8` instead of `UTF_8` as you suggested. Just want to be consistent how the current framework does it.